SimpleAdjustableFairings/ModuleSimpleAdjustableFairing.cs

```diff
@@ -120,6 +120,8 @@ public class ModuleSimpleAdjustableFairing : PartModule, IScalarModule, IPartCoM
         private bool needsRecalculateDragCubes;
         private bool needsNotifyFARToRevoxelize;
 
+		private float fairingScale = 1.0f;
```

Comment:
seems like weird indentation here?

Reply:
Yeah, I used tabs for intendation, as I got used to it. Sorry for breaking code conventions.